dex-load/sc1.Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+pipeline {
+    agent {
+        label 'buildagent-matcher'
+    }
+    options {
+        ansiColor('xterm')
+    }
+    parameters {
+        string(name: 'SBT_THREAD_NUMBER', defaultValue: '6', description: '')
+        string(name: 'SEED', defaultValue: 'test-seed', description: '')
+        string(name: 'AN', defaultValue: '60', description: '')
+        string(name: 'RC', defaultValue: '10000', description: '')
+        string(name: 'RT', defaultValue: '6', description: '')
+        string(name: 'AS', defaultValue: 'D', description: '')
```

Comment:
Let's write a description for each parameter. Now it is not clear, what is AN, RC, ...

Reply:
fixed